deku/crypto/crypto.rei

```diff
@@ -19,4 +19,5 @@ module BLAKE2B_20: (module type of BLAKE2B_20);
 module Secret: (module type of Secret);
 module Key: (module type of Key);
 module Key_hash: (module type of Key_hash);
+module Operation_hash: (module type of BLAKE2B_20);
```

Comment:
this shouldn't be just blake2B_20, tezos has `o` prefix, and for contract address it's not just blake2b_20, it's blake2B_20.hash of blake2b_20.t